scripts/SMD_chip_package_rlc-etc/size_definitions/size_capacitor_chip.yaml

```diff
@@ -0,0 +1,143 @@
+C_0504:
+    code_imperial: "0504"
+    code_metric: "1310"
+    body_length:
+        minimum: 1.02
+        maximum: 1.32
+    body_width:
+        minimum: 0.77
+        maximum: 1.27
+    terminal_length:
+        minimum: 0.13
+        maximum: 0.38
+    size_info: '(Body size source: IPC-SM-782 page 76, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+C_0603:
+    code_imperial: "0603"
+    code_metric: "1608"
+    body_length:
+        minimum: 1.45
+        maximum: 1.75
+    body_width:
+        minimum: 0.65
+        maximum: 0.95
+    terminal_length:
+        minimum: 0.2
+        maximum: 0.5
+    size_info: '(Body size source: IPC-SM-782 page 76, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+C_0805:
+    code_imperial: "0805"
+    code_metric: "2012"
+    body_length:
+        minimum: 1.8
+        maximum: 2.2
+    body_width:
+        minimum: 1.05
+        maximum: 1.45
+    terminal_length:
+        minimum: 0.25
+        maximum: 0.55 # only deviation from IPC is reducing this from 0.75mm
+    size_info: '(Body size source: IPC-SM-782 page 76, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf, https://docs.google.com/spreadsheets/d/1BsfQQcO9C6DZCsRaXUlFlo91Tg2WpOkGARC1WS5S8t0/edit?usp=sharing)'
+
+C_1206:
+    code_imperial: "1206"
+    code_metric: "3216"
+    body_length:
+        minimum: 3
+        maximum: 3.4
+    body_width:
+        minimum: 1.4
+        maximum: 1.8
+    terminal_length:
+        minimum: 0.25
+        maximum: 0.75
+    size_info: '(Body size source: IPC-SM-782 page 76, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+C_1210:
+    code_imperial: "1210"
+    code_metric: "3225"
+    body_length:
+        minimum: 3
+        maximum: 3.4
+    body_width:
+        minimum: 2.3
+        maximum: 2.7
+    terminal_length:
+        minimum: 0.25
+        maximum: 0.75
+    size_info: '(Body size source: IPC-SM-782 page 76, https://www.pcb-3d.com/wordpress/wp-content/uploads/ipc-sm-782a_amendment_1_and_2.pdf)'
+
+C_1812:
+    code_imperial: "1812"
+    code_metric: "4532"
+    body_length:
+        minimum: 4.2
+        maximum: 4.8
+    body_width:
+        minimum: 3
+        maximum: 3.4
+    terminal_length:
+        minimum: 0.25
+        maximum: 0.95
+    size_info: '(Body size source: https://www.nikhef.nl/pub/departments/mt/projects/detectorR_D/dtddice/ERJ2G.pdf)'
```

Comment:
Why did you chose that datasheet? It is also in the one from AVX and in IPC-SM-782.
IPC has which is identical, So I suggest to take that value.
```yaml
body_length: 4.2 .. 4.8
body_width: 3.0 .. 3.4
terminal_length: 0.25 .. 0.95
```

Reply:
That was the datasheet already there, and I don't know why. I also confirmed the values are in IPC-SM-782 and updated the link so that used for other chip Cs.